libcxx/test/std/utilities/tuple/tuple.tuple/tuple.apply/make_from_tuple.pass.cpp

```diff
@@ -195,6 +195,34 @@ void test_noexcept() {
     }
 }
 
+namespace LWG3528 {
+template <class _Tp, class _Tuple, class = std::void_t<>>
+struct can_make_from_tuple : std::false_type {};
+template <class _Tp, class _Tuple>
+struct can_make_from_tuple<
+    _Tp,
+    _Tuple,
+    std::void_t<decltype(std::__make_from_tuple_impl<_Tp>(
```

Comment:
I'm not too happy with the implementation details. I like to see whether we can use SFINAE.

Reply:
done